.travis.yml

```diff
@@ -4,18 +4,31 @@ os:
 
 language: csharp
 
+services:
+  - docker
+
 env:
   - CONFIGURATION=Release
 
 mono:
-  - 4.8.0
+  - latest
+  - 4.8
+
+matrix:
+  exclude:
+    - os: osx
+      mono: latest
+    - os: linux
+      mono: 4.8
 
 solution: IronPython.sln
 
 script:
-  - if [[ "$TRAVIS_OS_NAME" != "osx" ]] ; then sudo apt-get install -y binfmt-support libmono-system-windows-forms4.0-cil ; fi
-  - make
-  - make test-all
+  - docker 
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]] ; then make ; fi
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]] ; then docker run mono:4.8 -v /home/travis/build/IronLanguages/ironpython2:/home/travis/build/IronLanguages/ironpython2 /bin/sh -c "apt-get update ; apt-get install make ; cd /home/travis/build/IronLanguages/ironpython2 && make test-all" ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]] ; then docker run mono:5.0 -v /Users/travis/build/IronLanguages/ironpython2:/home/travis/build/IronLanguages/ironpython2 /bin/sh -c "cd /home/travis/build/IronLanguages/ironpython2 && make" ; fi
```

Comment:
OSX puts the user directories in /Users. I checked the output on a previous Travis job on OSX